### PR TITLE
paper review

### PR DESCRIPTION
**Pull request by jachermocilla**

### CODE REVIEW

**Comment by jachermocilla** [comment]
Following up on this pull request..

File: content/articles/jach/jach_004.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Lock–Unlock: Is That All? A Pragmatic Analysis of Locking in Software Systems
+###############################################################################
+
+:date: 2019-4-14
+:category: Paper Review
+:author: Joseph Anthony C. Hermocilla
+
+Most processors nowadays are multi-core processors to take advantage of parallel processing. 
+Associated with parallel processing is the synchronization problem wherin locks are extensively
```
Comment (mdqueliste) [inline content/articles/jach/jach_004.rst]: *wherein

File: content/articles/jach/jach_004.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+used. This paper [GUERRAOUI2019]_ argues that despite the large amount of available choices for 
+lock algorithms, developers do not have a comprehensive guide to select an algorithm. The paper also 
+asserts that most research on lock algorithms are limited to mutex locks and rarely consider other 
+techniques like trylocks and condition variables. In addition, most evaluation of lock algorithms 
+use workload that are specific to the design of the lock. Lastly, evaluation is focused only on throughput 
+neglecting energy efficiency and tail latency.
+
+The main contribution of this paper is a broad performance study of lock algorithms. Specifically, 
+28 state-of-the-art algorithms were studied and applied to 40 applications. The experiments where 
+conducted of 4 different multi-core machines. As mentioned, this study considered energy efficiency and
```
Comment (mdqueliste) [inline content/articles/jach/jach_004.rst:19]: *on 4 different multi-code machines

File: content/articles/jach/jach_005.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+:author: Joseph Anthony C. Hermocilla
+
+
+It is quite difficult for users to select VM configurations to optimize the 
+performance of their applications, especially if there are a lot of choices. 
+For example, if I want to run computational chemistry app, I need to make a decision 
+whether to use a VM with 1 VCPU and 2GB RAM or a VM with 2 VCPUs and 4GB RAM. 
+
+Cloud providers run benchmarks to measure the performance of various VM configurations for applications. 
+However, these benchmarks are done independent of the application to be deployed. Most like because
```
Comment (mdqueliste) [inline content/articles/jach/jach_005.rst]: * Most likely?

File: content/articles/jach/jach_005.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+It is quite difficult for users to select VM configurations to optimize the 
+performance of their applications, especially if there are a lot of choices. 
+For example, if I want to run computational chemistry app, I need to make a decision 
+whether to use a VM with 1 VCPU and 2GB RAM or a VM with 2 VCPUs and 4GB RAM. 
+
+Cloud providers run benchmarks to measure the performance of various VM configurations for applications. 
+However, these benchmarks are done independent of the application to be deployed. Most like because 
+providers have no knowledge of what application will be deployed. ]
```
Comment (mdqueliste) [inline content/articles/jach/jach_005.rst]: Remove ]

File: content/articles/jach/jach_004.rst (hunk 2 above)
Comment (jachermocilla) [reply content/articles/jach/jach_004.rst:19]: multi-core - because multiple CPU cores

**Pull request merged**